doc/sphinx/yaml/phases.rst

```diff
@@ -94,6 +94,8 @@ and optionally reactions that can take place in that phase. The fields of a
     - :ref:`pure-fluid <sec-yaml-pure-fluid>`
     - :ref:`Redlich-Kister <sec-yaml-Redlich-Kister>`
     - :ref:`Redlich-Kwong <sec-yaml-Redlich-Kwong>`
+    - :ref:`Peng-Robinson <sec-yaml-Peng-Robinson>`
+    - :ref:`plasma <sec-yaml-plasma>`
```

Comment:
This list was in alphabetical order at one point, but in addition to these phases, `electron-cloud` is out of order (because it was renamed from `metal`).